cmp/grid/Grid.js

```diff
@@ -193,13 +194,12 @@ class LocalModel {
     }
 
     createDefaultAgOptions(props) {
-        const {model} = this,
-            {useDeltaSort, useTransactions} = model.experimental;
+        const {model} = this;
 
         let ret = {
             model: model.agGridModel,
-            deltaSort: useDeltaSort && !model.treeMode,
-            immutableData: !useTransactions,
+            immutableData: true,
+            rowDataChangeDetectionStrategy: 'IdentityCheck',
```

Comment:
Looks like this is the default value when `immutableData: true` - are we concerned that ag-grid won't always respect that defaulting? Might be better to let ag-grid use whatever it thinks is best with immutable data, especially since this is grid property is pretty special and not really exposed in the standard grid options

Reply:
Yea, was trying to just get all the advanced options that might be useful for us in our code, so we could pull all the levers independently.  I will put a comment there, or remove it.

Note we are changing immutableData to *true* but I think that is the right thing to do.  From what I can see,  the *only* thing that immutableData does right now is set the changeDetectionStrategy for when rowData is changed in bulk (which we never do now anyway because we use transactions.).  

We could have considered removing our use of own transactions but to me that still seems the most prudent -- its a supported part of the API -- we can create it efficiently, and we also use part of it for our isRendererComplex bit.

Reply:
👍